pyro/poutine/trace_poutine.py

```diff
@@ -129,13 +129,13 @@ def __exit__(self, *args, **kwargs):
                 get_vectorized_map_data_info(self.trace)
         return super(TraceMessenger, self).__exit__(*args, **kwargs)
 
-    def get_trace(self, *args, **kwargs):
```

Comment:
This does not seem to be the latest `dev`; could you merge and update the PR?

```
    def get_trace(self, *args, **kwargs):
        """
        :returns: data structure
        :rtype: pyro.poutine.Trace

        Helper method for a very common use case.
        Calls this poutine and returns its trace instead of the function's return value.
        """
        self(*args, **kwargs)
        return self.msngr.get_trace()
```

Reply:
@neerajprad This change happens at the messenger class, not the poutine class. I use the dev branch.